tools/c7n_azure/c7n_azure/resources/appserviceplan.py

```diff
@@ -56,14 +56,16 @@ class ResizePlan(AzureBaseAction):
               resource: azure.appserviceplan
               actions:
                - type: resize-plan
-                 size: F1 # F1, D1, S1, S2, S3, P1, P2, P3
+                 size: F1
                  count: 1
     """
 
     schema = utils.type_schema(
         'resize-plan',
         **{
-            'size': {'type': 'string', 'enum': ['F1', 'D1', 'S1', 'S2', 'S3', 'P1', 'P2', 'P3']},
+            'size': {'type': 'string', 'enum':
+                    ['F1', 'B1', 'B2', 'B3', 'D1', 'S1', 'S2', 'S3', 'P1', 'P2',
+                     'P3', 'P1V2', 'P2V2', 'P3v2', 'PC2', 'PC3', 'PC4']},
```

Comment:
should this also have FREE, SHARED, BASIC, etc. else why is it in the sku name resolver if a user can't specify it?

Reply:
err - I was going back and forth between supporting those simplified names which the CLI supports, FREE==F1, SHARED==D1, etc and left it in-between.  Will clean that up.